config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,25 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
```

Comment:
I don't think so. This is just auto-generated by the `kubebuilder` command in case we want to leverage webhooks. I can remove it from the project when we move to v1.0.